projects/angular/src/progress/spinner/STYLES.md

```diff
@@ -1,5 +1,21 @@
 # Spinner
 
+## CSS Custom Properties
+
+| CSS Custom Property                    | Description                                  |
+| -------------------------------------- | -------------------------------------------- |
+| --clr-spinner-default-size             | Size (diameter) of large spinner             |
+| --clr-spinner-medium-size              | Size of medium spinner                       |
+| --clr-spinner-small-size               | Size of small spinner                        |
+| --clr-spinner-spacing-to-text          | Spacing between spinner and text next to it  |
```

Comment:
Can you rename this to `--clr-spinner-margin-right`?
There may be anything on the right not only text.